src/lib.rs

```diff
@@ -308,8 +306,8 @@ impl serde::Serialize for ISIN {
     where
         S: serde::Serializer,
     {
-        // TODO we could decide to serialize as bytes here, using `serializer.is_human_readable()`
-        serializer.serialize_str(self.value())
+        // TODO we could decide to serialize as byte array here, using `serializer.is_human_readable()`
```

Comment:
Let's remove this comment. We can have an "issue" filed that proposes support for byte array serialization and we can discuss in that issue what it would look like to use that vs string serialization as a user of the crate.